fdbserver/DDTxnProcessor.actor.cpp

```diff
@@ -340,11 +341,11 @@ class DDTxnProcessorImpl {
 					std::sort(dataMove->primaryDest.begin(), dataMove->primaryDest.end());
 					std::sort(dataMove->remoteDest.begin(), dataMove->remoteDest.end());
 
-					auto ranges = result->dataMoveMap.intersectingRanges(meta.range);
+					auto ranges = result->dataMoveMap.intersectingRanges(meta.ranges.front());
```

Comment:
what happens to the rest of ranges inside `meta.ranges`? Do they need to be processed and inserted into `result->dataMoveMap` as well?

Is it because right now the size of `meta.ranges` can only be 1, but you plan to add more later? If so, maybe we should assert the size is 1 so that in the future, we can easily fail the test when adding more to `meta.ranges` to remind us changes are needed at the assertions.

Reply:
Yes, currently we expect only 1, added the ASSERT.